### DISCUSSION TITLE
APIエラー

### DISCUSSION THREAD

**Discussion by kurochan** [topotal/waroom-roadmap#8] [Bug Report]
https://api.beta.waroom.com/graphql が500エラーを返却しており、トップページが画像のようになっていました。
![image](https://user-images.githubusercontent.com/591247/149932284-aff6860a-d657-4b0f-96ed-2055ce2f086c.png)

**Comment by kurochan** [comment]
すみません、こちら自己解決しました 🙇 
新規登録したあと、Invitation Link(Waroom Admin があなたを xxxxxxx に招待しています。 というメール)を踏む前にログインしてしまうとこのような状態に陥ってしまうようでした。

Invitation Linkをクリックしてからログインしなおすと治りました。

**Reply by sawa-zen** [reply]
@kurochan 
バグ報告ありがとうございます！詳細な手順まで書いていただき大変助かります 🙇🏻

**Reply by sawa-zen** [reply]
https://github.com/topotal/waroom-roadmap/issues/9

**Comment by sawa-zen** [comment]
こちら解決済みなのでクローズします。
